recipes-bsp/rpi-eeprom/rpi-eeprom_git.bb

```diff
@@ -0,0 +1,71 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
+
+DESCRIPTION = "This repository contains the rpi4 bootloader and scripts \
+for updating it in the spi eeprom"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=7dcd1a1eb18ae569857c21cae81347cb"
+
+# EEPROM configuration default values for this version, as specified at
+# https://www.raspberrypi.org/documentation/hardware/raspberrypi/booteeprom.md
```

Comment:
I'm not sure what is this referring to. The next line is not a configuration - it's the source's repository.